solutions/run-multicluster-servicemesh/README.md

```diff
@@ -0,0 +1,142 @@
+# Set up a Multicluster Service Mesh on OCM
+
+This scripts is to setup an multicluster service mesh on top of OCM. The guide will bootstrap 3 kind clusters(hub, cluster1, and cluster2) on your local machine and then deploy the [multicluster mesh addon](https://github.com/open-cluster-management-io/multicluster-mesh). After that, creating service meshes from the hub to the managed clusters and finally federating the service mshes so that microservices deployed into different managed clusters can be access each other.
+
+## Prerequisite
+
+Set up the dev environment in your local machine following [setup dev environment](../setup-dev-environment).
+
+## Install Multicluster Service Mesh Addon on OCM
+
+1. Clone the multicluster-mesh github repository and install:
+
+```bash
+git clone git@github.com:open-cluster-management-io/multicluster-mesh.git && cd multicluster-mesh
```

Comment:
I think we should build a helm chart and push to ocm helm repo, so we do not need to clone the code.

Reply:
filed a new PR for this: https://github.com/open-cluster-management-io/helm-charts/pull/7

Reply:
I think this can be done by adding a github action in mesh repo?

Reply:
thanks @qiujian16 
I updated the [PR](https://github.com/open-cluster-management-io/multicluster-mesh/pull/5/commits/e9af0c8fefa3f24abc2ec7fe4557c03c97c354cd) for the mesh repo to add release workflow, that workflow will upload teh chart tarball to the `helm-charts` repo.

Reply:
Updated installation with helm chart from `https://openclustermanagement.blob.core.windows.net/releases/`